### PR TITLE
ADAPT-2296: PurgeCSS Safe Code

### PR DESCRIPTION
**Pull request by sherakama**
# READY FOR REVIEW

# Summary
- Refactors components to eliminate dynamic classes

# Needed By (Date)
- End of sprint

# Urgency
- 5/10

# Steps to Test

1. Review the code 
2. Navigate to the preview build and test out the options to make sure they all still work (Flex, Grid, Heading)
3. Pull this branch into saa_alumni and build using the JIT compiler (untested right now)

# Affected Projects or Products
- OOD Online Giving
- SAA Alumni

# Associated Issues and/or People
- ADAPT-2296
- @jdwjdwjdw FYI. 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by yvonnetangsu** [comment]
Yay it works - thank you for all this work
https://deploy-preview-21--stanford-alumni.netlify.app/test/yvonne/test-section/

File: src/Grid/Grid.levers.js [hunk 1]
```diff
@@ -1,14 +1,110 @@
...
+export const gridCols = {
+  'xs': {
+    '1': 'su-grid-cols-1',
+    '2': 'su-grid-cols-2',
+    '3': 'su-grid-cols-3',
+    '4': 'su-grid-cols-4',
+    '5': 'su-grid-cols-5',
+    '6': 'su-grid-cols-6',
+    '7': 'su-grid-cols-7',
+    '8': 'su-grid-cols-8',
```
Comment (yvonnetangsu) [inline src/Grid/Grid.levers.js:34]: At first I was a little worried that these might not work if an actual number is entered as a prop, e.g., xs={3}, but it seems to work for both number and string. That's great!

**Review by yvonnetangsu — APPROVED** [review]
I think you got them all. Thank you for the work - much appreciated. Tested on the SAA branch. Seems to work perfectly.
https://deploy-preview-21--stanford-alumni.netlify.app/test/yvonne/test-section/

**Comment by yvonnetangsu** [comment]
I guess we could open another PR to use JIT for this repo then?

**Comment by sherakama** [comment]
> I guess we could open another PR to use JIT for this repo then?

Baby steps. This will unblock us on the saa_alumni work and we can look into safelist and JIT compiling for storybook later.

**Comment by yvonnetangsu** [comment]
> > I guess we could open another PR to use JIT for this repo then?
> 
> Baby steps. This will unblock us on the saa_alumni work and we can look into safelist and JIT compiling for storybook later.

This is a pretty big step already - thanks for all that refactoring. Ready for me to merge?

**Comment by yvonnetangsu** [comment]
Cuuute 😄

**Pull request merged**